examples/07_advanced/plot_advanced_decoding_scikit.py

```diff
@@ -0,0 +1,203 @@
+"""
+Advanced decoding using scikit learn
+==========================================
+
+This tutorial opens the box of decoding pipelines to bridge integrated
+functionalities provided by the :class:`nilearn.decoding.Decoder` object
+with more advanced usecases. It reproduces basic examples functionalities with
+direct calls to scikit-learn function and gives pointers to more advanced
+objects. If some concepts seem unclear, please refer to the :ref:`documentation
+on decoding <decoding_intro>` and in particular to the
+:ref:`advanced section <going_further>`. As in many other examples,
+we perform decoding of the visual category of a stimuli on Haxby
+2001 dataset, focusing on distinguishing two categories : face and cat images.
+
+    * J.V. Haxby et al. "Distributed and Overlapping Representations of Faces
+      and Objects in Ventral Temporal Cortex", Science vol 293 (2001), p
+      2425.-2430.
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+"""
+
+###########################################################################
+# Retrieve and load the fMRI data from the Haxby study
+# ------------------------------------------------------
+#
+# First download the data
+# ........................
+#
+
+# The :func:`nilearn.datasets.fetch_haxby` function will download the
+# Haxby dataset composed of fmri images in a Niimg, a spatial mask and a text
+# document with label of each image
+from nilearn import datasets
+haxby_dataset = datasets.fetch_haxby()
+mask_filename = haxby_dataset.mask_vt[0]
+fmri_filename = haxby_dataset.func[0]
+# Loading the behavioral labels
+import pandas as pd
+behavioral = pd.read_csv(haxby_dataset.session_target[0], delimiter=' ')
```

Comment:
Maybe we should use sphinx-gallery a bit more here, and display the "behavioral" dataframe
```suggestion
behavioral = pd.read_csv(haxby_dataset.session_target[0], delimiter=' ')
behavioral

###########################################################################
```